docker/sgx/Dockerfile

```diff
@@ -6,6 +6,7 @@ RUN apt-get update && \
     apt-get install -y tar && \
     apt-get install -y curl && \
     apt-get install -y open-enclave=0.19.4 && \
+    apt-get install -y clang-tools && \
```

Comment:
Does adding this affect at all the result of the normal SGX builds? Do the MRENCLAVE values, for example, get altered?

Reply:
I don't see how it would affect the build, since we use `gcc`, but I'll double-check just in case.